lib/templ.ml

```diff
@@ -178,7 +178,12 @@ and eval_simple_variable conf =
       in
       Util.doctype conf ^ "\n"
   | "highlight" -> conf.highlight
-  | "image_prefix" -> Util.image_prefix conf
+  | "image_prefix" ->
+      if conf.cgi then
+        match Util.p_getenv conf.base_env "image_prefix" with
+        | Some x -> x
+        | None -> conf.image_prefix
```

Comment:
Maybe we should still use the Util.image_prefix call in that case and the non cgi one, it avoids having to change more lines of code if we ever change the conf record at some point.